File: curso-python/ex032.py
# faça um programa que leia um ano qualquer e mostre se ele é bissexto
ano = int(input("digite um ano qualquer: "))

#Um ano é bissexto se:
#1. É divisível por 4
#e
#Não é divisível por 100
#OU
#É divisível por 400
if (ano % 4 == 0) and (ano % 100 != 0) or (ano % 400 == 0):
    print(f"{ano} é bissexto")
else:
    print(f"{ano} não é bissexto")

File: javascript-exercicios/ex032.js
const prompt = require("prompt-sync")();

const ano = parseInt(prompt("Digite um ano qualquer: "));

// Verifica se é bissexto
if ((ano % 4 === 0 && ano % 100 !== 0) || (ano % 400 === 0)) {
    console.log(`${ano} é bissexto`);
} else {
    console.log(`${ano} não é bissexto`);
}
